### PR TITLE
Expandable list and table - Added Helper methods.

### PR DESCRIPTION
**Pull request by silvergrund**
Expandable list view with logic fixed
Expandable table view with logic fixed
Table headers align with expandable content
Layout Overhaul to go with styleguide
Translation

### CODE REVIEW

**Review by Muckbuck — CHANGES_REQUESTED** [review]

File: source/js/Front/Components/Accordion.js [hunk 1]
```diff
@@ -1,34 +1,49 @@
...
+
             }
+
             {Object.keys(items).length === 0 &&
             <div className="gutter"><p>{translation.noResults}</p></div>
             }
-            {items.map(item => (
+
+            {view === 'accordiontable' &&
+            <header className="c-accordion__button-wrapper">
```
[the remark this answers is not in the document]
Comment (silvergrund) [reply source/js/Front/Components/Accordion.js]: Should be resolved...

File: source/js/Front/Components/Accordion.js [hunk 2]
```diff
@@ -1,34 +1,49 @@
...
+            className="c-accordion"
+            js-expand-container=""
+            data-uid="5fce30f28f9d0">
+
             {showSearch &&
-                <SearchField
-                    doSearch={doSearch}
-                    translation={translation}
-                />
+            <SearchField
```
[the remark this answers is not in the document]
Comment (silvergrund) [reply source/js/Front/Components/Accordion.js]: Should be resolved...

File: source/js/Front/Components/Accordion.js [hunk 3]
```diff
@@ -1,34 +1,49 @@
...
+
             {Object.keys(items).length === 0 &&
             <div className="gutter"><p>{translation.noResults}</p></div>
             }
-            {items.map(item => (
+
+            {view === 'accordiontable' &&
+            <header className="c-accordion__button-wrapper">
+                {fieldMap.heading.map((heading, i) => (
+                    <span key={i}
```
[the remark this answers is not in the document]
Comment (silvergrund) [reply source/js/Front/Components/Accordion.js]: Should be resolved...

File: source/js/Front/Components/Accordion.js [hunk 4]
```diff
@@ -1,34 +1,49 @@
...
+            }
+
+            {items.map((item, index) => (
+
                 <AccordionItem
                     key={item.id}
                     heading={item.heading}
                     content={item.content}
                     view={view}
+                    index={index + '-' + Math.round(Math.random() * 100000000)}
```
[the remark this answers is not in the document]
Comment (silvergrund) [reply source/js/Front/Components/Accordion.js]: Should be resolved...

File: source/js/Front/Components/AccordionItem.js [hunk 5]
```diff
@@ -1,32 +1,67 @@
...
-        <button tabIndex="0" className="accordion-toggle c-accordion__button u-padding__x--3 u-padding__y--2" htmlFor="accordion-section-1">
-            {heading[0]}
-        </button>
-        }
 
         {view === 'accordiontable' &&
-        <button tabIndex="0" className="accordion-toggle c-accordion__button u-padding__x--3 u-padding__y--2" htmlFor="accordion-section-1">
-                <span className="accordion-table">
-                {heading.map((title, i) => (
-                    <span key={i} className="column-header" dangerouslySetInnerHTML={{__html: title}} />
```
Comment (alexanderbomanskoug2) [inline source/js/Front/Components/AccordionItem.js:13]: Key is not unique

File: source/js/Front/Components/AccordionItem.js [hunk 6]
```diff
@@ -1,32 +1,67 @@
...
+                onClick={itemClicked.bind(this)}
+                className="c-accordion__button column-headings-flex"
+                aria-controls={'c-accordion__aria-jasonRender-' + index}
+                aria-expanded="false"
+                js-expand-button=""
+                htmlFor={'c-accordion__aria-jasonRender-' + index}>
+                <span className="c-accordion__button-wrapper">
+                    {heading.map((title, i) => (
+
+                        <span key={i}
```
Comment (alexanderbomanskoug2) [inline source/js/Front/Components/AccordionItem.js:15]: Key is not unique

File: source/js/Front/Components/AccordionItem.js [hunk 7]
```diff
@@ -1,32 +1,67 @@
...
+        }
+
+        <div
+            className="c-accordion__content"
+            id={'c-accordion__aria-jasonRender-' + index}
+            aria-hidden="true">
+
             {content.filter(section => section.value).map((section, i) => (
-                <div key={i} className="u-mb-2">
+                <p key={i} className="u-mb-2">
```
Comment (alexanderbomanskoug2) [inline source/js/Front/Components/AccordionItem.js:54]: Key is not unique

File: source/js/Front/Components/Table.js [hunk 8]
```diff
@@ -3,38 +3,38 @@ import SearchField from './SearchField';
...
-            <tbody class="c-table__body">
-            {items.map(item => (
-                <tr class="c-table__line" key={item.id}>
-                    {item.heading.map((heading, i) => (
-                        <td class="c-table__column" key={i} dangerouslySetInnerHTML={{__html: heading}} />
+            <table class="c-table__table">
+                <thead class="c-table__head">
+                <tr class="c-table__line">
+                    {fieldMap.heading.map((heading, i) => (
+                        <th class="c-table__column" key={i}>
```
Comment (alexanderbomanskoug2) [inline source/js/Front/Components/Table.js:21]: Key is not unique

File: source/js/Front/Components/Table.js [hunk 9]
```diff
@@ -3,38 +3,38 @@ import SearchField from './SearchField';
...
-            ))}
-            </tbody>
-        </table>
+                </thead>
+
+                <tbody class="c-table__body">
+                {items.map(item => (
+                    <tr class="c-table__line" key={item.id}>
+                        {item.heading.map((heading, i) => (
+                            <td class="c-table__column" key={i} dangerouslySetInnerHTML={{__html: heading}} />
```
Comment (alexanderbomanskoug2) [inline source/js/Front/Components/Table.js:32]: Key is not unique

**Review by alexanderbomanskoug2 — CHANGES_REQUESTED** [review]

**Comment by alexanderbomanskoug2** [comment]
Changes requested but not required, otherwise approved

**Review by silvergrund — COMMENTED** [review]
> Changes requested but not required, otherwise approved

Out of scope - Not a refactor commit

**Pull request merged**